### PR TITLE
variables table

### PR DESCRIPTION
**Pull request by davidplumb**

### CODE REVIEW

File: tables/variables.md [hunk 1]
```diff
@@ -1 +1,71 @@
+# tables/variables
 
+The `Variables` table displays information concerning the various different `variables` that are stored on the database.
+The following `JOIN` queries can be carried out:
+
+- `geography_type_id` on the [topics](topics.md) table using `topic_id`.
+
+## What are top-level variables?
```
Comment (tomgiddingsjisc) [inline tables/variables.md]: Removal of top-level

File: tables/variables.md [hunk 2]
```diff
@@ -1 +1,71 @@
+# tables/variables
 
+The `Variables` table displays information concerning the various different `variables` that are stored on the database.
+The following `JOIN` queries can be carried out:
+
+- `geography_type_id` on the [topics](topics.md) table using `topic_id`.
```
Comment (tomgiddingsjisc) [inline tables/variables.md]: Does this mean to say `geography_type_id`?

File: tables/variables.md [hunk 3]
```diff
@@ -1 +1,71 @@
...
+
+`variables` relate to topics in a many to one relationship. Where every [topic](topics.md) is sub-divided into many different variables. These `variables` alongside their parent `topics` represent context of the database as without them the data would just be arbitrary numbers alongside geographical locations.
+
+## Example use
+
+I want to identify every child `variable` for the `topic` `SEX`. We will start by identidying the `id` for the chosen `topic` of `SEX`.
+
+```sql
+SELECT id, abbreviation, description  FROM c2011_meta.topics 
+WHERE abbreviation = SEX;
```
Comment (tomgiddingsjisc) [inline tables/variables.md]: This query doesn't run (missing quotes) - definitely worth running these quickly just as a sanity check.

**Review by tomgiddingsjisc — COMMENTED** [review]
Just a few small changes

File: tables/variables.md (hunk 3 above)
Comment (davidplumb) [reply tables/variables.md]: fair enough, idk how that ended up like that but good spot

File: tables/variables.md (hunk 2 above)
Comment (davidplumb) [reply tables/variables.md]: nope. fixed

File: tables/variables.md (hunk 1 above)
Comment (davidplumb) [reply tables/variables.md]: yep done

**Review by tomgiddingsjisc — APPROVED** [review]
Brill - looks great to me!

**Pull request merged**